### PR TITLE
Add web tool

### PR DESCRIPTION
**Pull request by jmhauck**
https://devtopia.esri.com/WebGIS/solution-deployment-apps/issues/100

### CODE REVIEW

**Comment by jmhauck** [comment]
@MikeTschudi lint and build work locally without reporting any issues. Not sure why its failing here

File: packages/web-tool/package.json [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    "@esri/arcgis-rest-auth": "^3.7.0",
+    "@esri/arcgis-rest-feature-layer": "^3.7.0",
+    "@esri/arcgis-rest-portal": "^3.7.0",
+    "@esri/arcgis-rest-request": "^3.7.0",
+    "@esri/arcgis-rest-service-admin": "^3.7.0",
+    "@esri/hub-common": "^14.109.1",
+    "@esri/hub-initiatives": "^14.0.0",
+    "@esri/hub-sites": "^14.2.4",
+    "@esri/hub-teams": "^14.1.0",
+    "@esri/solution-common": "^5.2.14",
```
Comment (MikeTschudi) [inline packages/web-tool/package.json:30]: Please change @esri/solution-* to version 5.2.16

File: packages/web-tool/src/web-tool-processor.ts [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+    }
+  });
+}
+
+/**
+ * Get the urls available on the portal self.
+ *
+ * @param {IItemTemplate} template The template
+ * @param {any} templateDictionary The template dictionary
+ * @param {UserSession} authentication The destination user session info
```
Comment (MikeTschudi) [inline packages/web-tool/src/web-tool-processor.ts:164]: Please update to "destinationAuthentication"

**Review by MikeTschudi — APPROVED** [review]

**Comment by MikeTschudi** [comment]
image

**Pull request merged**